app/routes/cdo/create/owner-summary.js

```diff
@@ -0,0 +1,34 @@
+const { routes, views } = require('../../../constants/owner')
+const { routes: dogRoutes } = require('../../../constants/dog')
+const { getOwnerDetails, getEnforcementDetails, getAddress } = require('../../../session/cdo/owner')
+const ViewModel = require('../../../models/cdo/create/owner-summary')
+const { admin } = require('../../../auth/permissions')
+const { getCourts, getPoliceForces } = require('../../../api/ddi-index-api')
+
+module.exports = [{
+  method: 'GET',
+  path: routes.ownerSummary.get,
+  options: {
+    auth: { scope: [admin] },
+    handler: async (request, h) => {
+      const ownerDetails = getOwnerDetails(request)
+      console.log('route ownerDetails', ownerDetails)
+      const address = getAddress(request)
+      const enforcementDetails = getEnforcementDetails(request)
+      const courts = await getCourts()
+      const policeForces = await getPoliceForces()
+
+      return h.view(views.ownerSummary, new ViewModel(ownerDetails, address, enforcementDetails, courts, policeForces))
+    }
+  }
+},
+{
+  method: 'POST',
+  path: routes.ownerSummary.post,
```

Comment:
Maybe something for a future ticket but would it be a good idea to validate the owner payload here?

Could end up with posting missing data if sections have been skipped.

Reply:
I agree, although it would make even more sense to validate the full payload at the point of record creation (e.g. at the dog summary page). Otherwise, the owner details might be ok hetre but then missing at the dog pages - somehow

Reply:
Good point we can look at this later. We are able to combine Joi schemas together so wouldn't harm doing the validation check here as well.